upgrade-guides/v4.0-v5.0.md

```diff
@@ -0,0 +1,37 @@
+# How to Upgrade Hazelcast-wm Version 4.0 to Version 5.0
```

Comment:
Are you going to add a link to this upgrade guide to the readme?

Reply:
good idea let me do it

Reply:
7cb0057